web-platform/website/src/components/pages/login/LoginPageContent.tsx

```diff
@@ -62,13 +84,18 @@ const styles = {
           maxWidth: "100%",
           height: "auto",
         },
+        ".login-col": {
+          marginRight: "3rem"
```

Comment:
Same here and below. IMO using custom values defeats the purpose of having enums that make the site more standardized. So 3 rem translates to 3 * 16 = 48px which translates to 1.2 * Spacing.xxxLarge I believe.

Reply:
All right, will keep in mind moving forward.